api/opds.py

```diff
@@ -768,6 +769,19 @@ def annotate_feed(self, feed, lane):
             )
             feed.add_link_to_feed(feed.feed, **crawlable_link)
 
+        if lane and lane.uses_customlists and (hasattr(lane, "customlist_ids") and lane.customlist_ids):
+            crawlable_url = self.url_for(
+                "crawlable_list_feed", list_name="custom list",
```

Comment:
I don't get what this is for. And why is the list name "custom list"?

Reply:
WorkList now has a `customlist_ids` property but a Lane does not. So checking to see if it's there in order to add the crawlable url. "custom list" is the name for now because I haven't been able to get the list name from just the array of custom list ids.

Reply:
There's a `customlist_ids` property but no `customlists` property? This is almost identical to the previous code except it doesn't restrict you to lanes with one list, but this probably should too. Right now you're just making a link that will give you a 404 since it's not a real list name.

Reply:
There could be either `customlist_ids` or `customlists`.
Right, I would have to do more to get the list from the id to get the name.